api/api/src/main/scala/uk/ac/wellcome/platform/api/services/ElasticsearchService.scala

```diff
@@ -41,15 +35,15 @@ class ElasticsearchService @Inject()(elasticClient: ElasticClient)(
   def listResults: (Index, ElasticsearchQueryOptions) => Future[
     Either[ElasticError, SearchResponse]] =
     executeSearch(
-      maybeQueryString = None,
+      maybeWorkQuery = None,
       sortDefinitions = List(fieldSort("canonicalId").order(SortOrder.ASC))
     )
 
   def simpleStringQueryResults(
```

Comment:
At this point `simpleStringQueryResults()` isn’t a very accurate name for this method, please change it.